packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -9716,6 +9716,29 @@ export default{
 
 			await runWrangler("deploy");
 		});
+		it("should correctly read sourcemaps with custom wrangler.toml location", async () => {
+			fs.mkdirSync("some/dir", { recursive: true });
+			writeWranglerToml(
+				{
+					main: "../../index.ts",
+					upload_source_maps: true,
+				},
+				"some/dir/wrangler.toml"
+			);
+			writeWorkerSource({ format: "ts" });
+
+			mockSubDomainRequest();
+			mockUploadWorkerRequest({
+				expectedMainModule: "index.js",
+				expectedModules: {
+					"index.js.map": expect.stringContaining(
+						`"sources":["../../another.ts","../../index.ts"],"sourceRoot":""`
```

Comment:
Can this test explicitly specify the `file` field of this? It's critical that it matches the name of a file in the multipart but has `.map` otherwise the stacktrace remapper in the runtime can't correlate module names to sourcemaps.

Reply:
(not that I think this PR would chance that field, but we should have been explicitly testing this in the first place)